### PR TITLE
mgr/influx: PEP-8 and other fixes to Influx module

### PR DESCRIPTION
**Pull request by wido**
This PR fixes two things:

- PEP-8 fixes to Influx module
- Send fsid as a tag to Influx
- Update module configuration
- Updated logging of module

By sending the fsid as a tag to Influx we can have multiple clusters send their data to the same Influx database.

### CODE REVIEW

**Comment by tchaikov** [comment]
@wido we add milestone for PR targeting non-master branches. i am assuming that you want to backport this change to luminous without an accompanied tracker ticket here, so added the "need-backport" label instead.

File: src/pybind/mgr/influx/module.py [hunk 1]
```diff
@@ -25,6 +25,9 @@ def __init__(self, *args, **kwargs):
         self.event = Event()
         self.run = True
 
+    def get_fsid(self):
+        return self.get('osd_map')['fsid']
```
Comment (jcsp) [inline src/pybind/mgr/influx/module.py]: I'd suggest using monmap here, just to save a few cycles as osdmap can be rather large.

File: src/pybind/mgr/influx/module.py [hunk 2]
```diff
@@ -55,6 +58,7 @@ def get_df_stats(self):
                         "pool_id" : pool['id'],
                         "type_instance" : df_type,
                         "mgr_id" : self.get_mgr_id(),
```
Comment (jcsp) [inline src/pybind/mgr/influx/module.py]: I just noticed this rogue `mgr_id` label here, let's remove this while we're updating labels

**Comment by jcsp** [comment]
Looks sane, just the two comments.

The mgr_id bit isn't your fault but we should make the update in this PR so that people don't get their series labels changing twice.

Can you also update the commit messages to say "mgr/influx" instead of "Mgr Influx"?  Weirdly they look right in some of the places in github but I think the actual commits still have the latter format.

**Comment by wido** [comment]
@jcsp Thanks! I've addressed your comments in my commits.

**Comment by wido** [comment]
@jcsp: While testing more with this module I wrote a bit more code and added commits to this PR.

Should make this module work a lot nicer.

**Pull request merged**